py4j-java/src/test/java/py4j/examples/ReturnerExample.java

```diff
@@ -46,7 +46,7 @@ public int computeInt(IReturnConverter returner) {
 	public int computeNothing(IReturnConverter returner) {
 		return 1;
 	}
-
+	
```

Comment:
useless new whitespace

Reply:
This pr has been merged.   Feel free to create pr to remove any unnecessary whitespace (in this pr or elsewhere).